proposals/NNNN-replace-pthread-implementation-with-threads-from-cpp11.md

```diff
@@ -0,0 +1,88 @@
+# Replace thread::Thread implementation with std::thread from C++11
```

Comment:
Can you please add code blocks around the threads for readability?

Reply:
added in 28259e42e9d61ba17695f593df0c63288d5a49c9